src/main/app/src/app/zaken/model/zaak.ts

```diff
@@ -42,8 +40,7 @@ export class Zaak {
     isVerlengd: boolean;
     redenVerlenging: string;
     duurVerlenging: string;
-    groep: Group;
-    behandelaar: User;
+    toekenning: ZaakToekenning;
```

Comment:
Dit is raar. Het object is een zaak en die heeft een behandelaar en een groep. Beide kunnen null zijn of niet. Misschien wil een een Toekenning object hebben voor het verwerken van de toekenningen, maar dat is een bewerking op zaak-niveau betekent de toekennings-member niets.

Reply:
Ik dacht het toekenning subobject nodig te zijn voor het samenwerken met het bestaande MedewerkerGroepComponent, maar daar heb ik wat mogelijkheden over het hoofd gezien. Opgelost in nieuwe commit.